<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lpr</class>
 <widget class="QMainWindow" name="lpr">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>890</width>
    <height>580</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>870</width>
    <height>553</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>lpr</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>881</width>
      <height>571</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>871</width>
      <height>551</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_file">
     <attribute name="title">
      <string>文件追踪</string>
     </attribute>
     <widget class="QLabel" name="label_image_file">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>600</width>
        <height>450</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(0, 0, 0, 255), stop:0.19397 rgba(0, 0, 0, 255), stop:0.202312 rgba(122, 97, 0, 255), stop:0.495514 rgba(76, 58, 0, 255), stop:0.504819 rgba(255, 255, 255, 255), stop:0.79 rgba(255, 255, 255, 255), stop:1 rgba(255, 158, 158, 255));</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_text1_file">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>图像显示区域</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_text4_file">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>121</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>车牌号</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="plate_edit_file">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>150</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="text">
       <string>粤AWAQMM</string>
      </property>
     </widget>
     <widget class="QPushButton" name="searchStart_but_file">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>500</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>开始搜索</string>
      </property>
     </widget>
     <widget class="QPushButton" name="searchStop_but_file">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>500</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>暂停搜索</string>
      </property>
     </widget>
     <widget class="QPushButton" name="path_but_file">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>40</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>设置</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_text3_file">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>40</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>搜索路径</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_text2_file">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>11</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>搜索进度</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="path_edit_file">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>70</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_text5_file">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>321</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>搜索信息</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QTextEdit" name="searchInfo_edit_file">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>350</y>
        <width>241</width>
        <height>141</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="previous_but">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>500</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>上一张</string>
      </property>
     </widget>
     <widget class="QPushButton" name="next_but">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>500</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>下一张</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="plate_processBar">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>10</y>
        <width>161</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QPushButton" name="plateInfo_clearBut_file">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>320</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>清空</string>
      </property>
     </widget>
     <widget class="QComboBox" name="maxdiff_box_file">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>320</y>
        <width>41</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
     </widget>
     <widget class="QCheckBox" name="checkBoxPla_file">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>121</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>格式检索</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTextEdit" name="operatePla_edit_file">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>190</y>
        <width>241</width>
        <height>61</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxPla_file">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>120</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>添加</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>删除</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>清空</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="operatePla_but_file">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>150</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
     <widget class="QLabel" name="searchInfo_label_file">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>320</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="image_label.qrc">:/image/image/green.png</pixmap>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxMSER_file">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>290</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>MSER</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxSOBEL_file">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>290</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>SOBEL</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxCOLOR_file">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>290</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>COLOR</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_text6_file">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>260</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>检测车牌算法</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_video">
     <attribute name="title">
      <string>视频追踪</string>
     </attribute>
     <widget class="QLabel" name="label_image_video">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>600</width>
        <height>450</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(0, 0, 0, 255), stop:0.19397 rgba(0, 0, 0, 255), stop:0.202312 rgba(122, 97, 0, 255), stop:0.495514 rgba(76, 58, 0, 255), stop:0.504819 rgba(255, 255, 255, 255), stop:0.79 rgba(255, 255, 255, 255), stop:1 rgba(255, 158, 158, 255));</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_text1_video">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>图像显示区域</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="plate_edit_video">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>150</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="text">
       <string>粤AWAQMM</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_text3_video">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>121</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>车牌号</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="operatePla_but_video">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>150</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_text2_video">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>11</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>搜索源选择</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxPla_video">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>121</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>格式检索</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTextEdit" name="operatePla_edit_video">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>190</y>
        <width>241</width>
        <height>61</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="plateInfo_clearBut_video">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>320</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>清空</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="searchInfo_edit_video">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>350</y>
        <width>241</width>
        <height>141</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="searchStart_but_video">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>500</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>开始搜索</string>
      </property>
     </widget>
     <widget class="QPushButton" name="searchStop_but_video">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>500</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>暂停搜索</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxPla_video">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>120</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>添加</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>删除</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>清空</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_text5_video">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>321</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>搜索信息</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioCam_but_video">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>41</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>摄像头</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioVid_but_video">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>41</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>视频</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxCam_video">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>40</y>
        <width>41</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="path_but_video">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>40</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>设置</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="path_edit_video">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>70</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QComboBox" name="maxdiff_box_video">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>320</y>
        <width>41</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
     </widget>
     <widget class="QSlider" name="videoSlider">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>499</y>
        <width>601</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="searchInfo_label_video">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>320</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="image_label.qrc">:/image/image/green.png</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="label_text4_video">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>260</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>检测车牌算法</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxSOBEL_video">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>290</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>SOBEL</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxMSER_video">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>290</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>MSER</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxCOLOR_video">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>290</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>COLOR</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="image_label.qrc"/>
 </resources>
 <connections/>
</ui>
